README.md

```diff
@@ -776,6 +776,7 @@ API | Description | Auth | HTTPS | CORS | Link |
 | Dailymotion | Dailymotion Developer API | `OAuth` | Yes | Unknown | [Go!](https://developer.dailymotion.com/) |
 | Open Movie Database | Movie information | `apiKey` | Yes | Unknown | [Go!](http://www.omdbapi.com/) |
 | Ron Swanson Quotes | Television | No | Yes | Unknown | [Go!](https://github.com/jamesseanwright/ron-swanson-quotes#ron-swanson-quotes-api) |
+| STAPI | Star Trek API | No | No | No | [Go!](http://stapi.co) |
```

Comment:
Can you please use a more descriptive Description? Something like "Information on all things Star Trek" would be more appropriate/useful.